lib/src/cartesian/components/BarGraphLabels.tsx

```diff
@@ -46,19 +52,19 @@ export const BarGraphLabels = ({
     // Bar Midpoints
     const barVerticalMidpoint =
       (chartInnerTopEdge + chartInnerBottomEdge + Number(y)) / 2;
-    const barHorizontalMidpoint = x - labelWidth / 2;
+    // Move the label left by half its width to properly center the text over the bar
+    // const barHorizontalMidpoint = x - labelWidth / 2;
```

Comment:
changing this behavior with removing `barHorizontalMidpoint` feels like it'll break the layout of existing charts?